scripts/copy_ios_push_file.js

```diff
@@ -177,4 +176,66 @@ module.exports = function(ctx) {
         });
     }
 
```

Comment:
but the idea is to move to 6.4.0 and cordova-ios 4.3.0, right? if we do so, then we don't need the hook at all.
Anyway, if the hook is working for CLI 5 and 6 I don't think we need to simplify it once we deprecate CLI 5